app/controllers/links_controller.rb

```diff
@@ -0,0 +1,40 @@
+class LinksController < ApplicationController
+  skip_before_action :require_login, :authorize, :session_expiry, :update_activity_time, :set_taxonomy, :set_gettext_locale_db, :only => :show
+
+  def show
+    url = external_url(type: params[:type], options: params)
+    redirect_to(url)
+  end
+
+  def external_url(type:, options: {})
+    case type
+    when 'manual'
+      documentation_url(options['section'], options)
+    when 'feed'
+      Setting['rss_url']
+    when 'wiki'
+      wiki_url(section: options['section'])
+    when 'chat'
+      'https://freenode.net'
+    when 'forums'
+      'https://community.theforeman.org'
+    when 'issues'
+      'https://projects.theforeman.org/projects/foreman/issues'
+    when 'vmrc'
+      'https://www.vmware.com/go/download-vmrc'
+    end
+  end
+
+  def documentation_url(section = "", options = {})
+    root_url = options[:root_url] || "https://theforeman.org/manuals/#{SETTINGS[:version].short}/index.html#"
+    if section.empty?
+      "https://theforeman.org/documentation.html##{SETTINGS[:version].short}"
```

Comment:
This particular page isn't really showing anything anymore and just redirecting to the quickstart guide. It doesn't respect the version so you end up at the latest stable quickstart. This was probably also an issue before but probably something to look at. Given you're just moving code, it's fine to do it in a later PR.